guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,317 @@
+
+![Section5](https://i.imgur.com/OwYGx2s.png)
+
+- Leave the Hard disk file type to `(VDI)` 
+
+![Section5](https://i.imgur.com/XPpGqxm.png)
+
+- Leave storage on physical hard disk to `(Dynamically allocated)` this will used the hard drive memory dinamically until it reach the max you gave it on the next step. 
+
+![Section5](https://i.imgur.com/YGrvBmV.png)
+
+- For this machine the minimun for Ubuntu so it does not give you problem set the File location and `size to (20.00GB)`
+
+![Section5](https://i.imgur.com/p2jIaGn.png)
+
+- After the Virtual Machine is created now you have to add the `ISO` that you download at the beginning 
+- Go to Storage section and click on top that it say `[Optical Drive]Empty`
+
+![Section6](https://i.imgur.com/8Rfq6nZ.png)
+
+- Then press `Choose disk image...`
+
+![Section6](https://i.imgur.com/Hbb1y7h.png)
+
+- Find the location where you download your ISO for Ubuntu Server.
+
+![Section6](https://i.imgur.com/JHKaWpk.png)
+
+- On this step make sure you add the network adapters to the server 
+
+- `Right click` the `ISP Ubuntu` and click `settings`
+
+![Section7](https://i.imgur.com/8w6rRfO.png)
+
+- On network adapter one leave it to `NAT`
+- Then go to Network Adapter 2 and change the network adapter to `WAN Environment` like the picture bellow.
+
+![Section7](https://i.imgur.com/EB1ajQl.png)
+![Section7](https://i.imgur.com/N2OBoxs.png)
+
+- On this part start the ISP ubuntu virtual box to start the setup. 
+
+- Click on the `ISP Ubuntu` then press `start`
+
+![Section8](https://i.imgur.com/wvUCwDs.png)
+
+## Ubuntu Server installation <a id="Second"></a>
+
+2. The setup of the ISP Ubuntu on the photos bellow
+
+- select you language
+
+![Section9](https://i.imgur.com/Cr4wj1w.png)
+
+- Press enter on `Install Ubuntu Server`
+
+![Section9](https://i.imgur.com/FCEeu3L.png)
+
+- Select the language again
+
+![Section9](https://i.imgur.com/cptpprK.png)
+
+- Select your location
+
+![Section9](https://i.imgur.com/G03SKC5.png)
+
+- Press `no` on the Configure the Keyboard
+
+![Section9](https://i.imgur.com/6x0pEuX.png)
+
+- Select Keyboard language
+
+![Section9](https://i.imgur.com/zQ2EeJk.png)
+
+![Section9](https://i.imgur.com/jsibCL2.png)
+
+- Select `enp0s3` which is the `NAT` this might vary on your PC
+
+![Section9](https://i.imgur.com/xN9m2js.png)
+
+- Set the `Hostname` of you VM machine to whatever you want
+
+![Section9](https://i.imgur.com/isHnTuW.png)
+
+- Set the full name for `new user`
+
+![Section9](https://i.imgur.com/SeJZZFJ.png)
+
+![Section9](https://i.imgur.com/mmDpJK9.png)
+
+- Then set your `Username` for your account
+
+![Section9](https://i.imgur.com/zuIqs7E.png)
+
+- Set the new user `password`
+
+![Section9](https://i.imgur.com/t4L1H5v.png)
+
+- verify password again
+
+![Section9](https://i.imgur.com/Ypjs1pV.png)
+
+- Select if you want to encrypt your home directory `I advice to select no for this test lab`
```

Comment:
change `I advice to select no for this test lab` to I advice to select `no` for this test lab.